### PR TITLE
Added the api callbacks for login and createuser.

### PR DESCRIPTION
**Pull request by darkracer**
I didn't know what to do with the sessionId so I'm just logging the it.

### CODE REVIEW

File: app/src/main/java/com/kanshu/kanshu/LoginActivity.java [hunk 1]
```diff
@@ -1,12 +1,33 @@
...
+import retrofit.RetrofitError;
+import retrofit.client.Response;
+import retrofit.http.Header;
 
 
 public class LoginActivity extends BaseActivity {
 
+    RestAdapter restAdapter = new RestAdapter.Builder().setEndpoint("https://kanshu-ds.herokuapp.com").setLogLevel(RestAdapter.LogLevel.FULL).build();
+
+    KanshuApi kanshuApi = restAdapter.create(KanshuApi.class);
```
Comment (intrications) [inline app/src/main/java/com/kanshu/kanshu/LoginActivity.java]: As recommended by the Retrofit library, can you extract this to make it a singleton used across the whole app? It shouldn't be created every time it is used.

File: app/src/main/java/com/kanshu/kanshu/LoginActivity.java [hunk 2]
```diff
@@ -35,4 +56,35 @@ public boolean onOptionsItemSelected(MenuItem item) {
 
         return super.onOptionsItemSelected(item);
     }
+
+    public void Login(View v){
+
+        HttpGet request = new HttpGet("https://kanshu-ds.herokuapp.com/login");
```
Comment (intrications) [inline app/src/main/java/com/kanshu/kanshu/LoginActivity.java]: I would think there would be a more Retrofit way of doing this:

e.g. 
http://futurestud.io/blog/android-basic-authentication-with-retrofit/
http://stackoverflow.com/questions/21370620/retrofit-post-request-w-basic-http-authentication-cannot-retry-streamed-http

File: app/src/main/java/com/kanshu/kanshu/SignupActivity.java [hunk 3]
```diff
@@ -7,19 +7,28 @@
...
+import retrofit.Callback;
+import retrofit.RestAdapter;
+import retrofit.RetrofitError;
+import retrofit.client.Response;
+
 
 public class SignupActivity extends BaseActivity {
 
+    RestAdapter restAdapter = new RestAdapter.Builder().setEndpoint("https://kanshu-ds.herokuapp.com").setLogLevel(RestAdapter.LogLevel.FULL).build();
+    KanshuApi kanshuApi = restAdapter.create(KanshuApi.class);
```
Comment (intrications) [inline app/src/main/java/com/kanshu/kanshu/SignupActivity.java]: Again, extract to singleton.

File: app/src/main/java/com/kanshu/kanshu/SignupActivity.java [hunk 4]
```diff
@@ -108,9 +117,34 @@ public void onSelectLogInPage(View clicked) {
...
+            public void success(String s, Response response) {
+                //Log.i("SignupActivity", s);
+        }
+
+            @Override
+            public void failure(RetrofitError error) {
+                Log.i("SignupActivity", "Epic Fail!");
+            }
+        });
         User userData = new User("Username", "Intermediate Level");
```
Comment (intrications) [inline app/src/main/java/com/kanshu/kanshu/SignupActivity.java]: This is starting the next Activity even if the signup call fails.

**Comment by fernandobuzi** [comment]
@darkracer, all good. ;) Just checking if you saw @intrications' comments above.

**Comment by fernandobuzi** [comment]
@arandr, please go through this one when you're available?

File: app/src/main/java/ApiHandler.java [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import com.kanshu.kanshu.KanshuApi;
```
Comment (arandr) [inline app/src/main/java/ApiHandler.java]: I'm not sure why you seem to have this file in both the com.kanshu.kanshu package and outside it?

**Comment by arandr** [comment]
@darkracer Aside from the note above everything looks good.

**Comment by fernandobuzi** [comment]
@darkracer hi! How fast can you check @arandr's comment? It would be great if we were able to merge this PR today.

**Comment by darkracer** [comment]
I think I can check it this afternoon.

Fernando Buzi <notifications@github.com> schreef op 3 april 2015 10:00:10 CEST:
>@darkracer hi! How fast can you check @arandr's comment? It would be
>great if we were able to merge this PR today.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/asm-products/kanshu-android/pull/30#issuecomment-89209512

-- 
Verzonden van mijn Android telefoon met K-9 Mail.

**Comment by fernandobuzi** [comment]
@darkracer, all well? I imagine you were off during Easter. Are you able to close this PR before Friday?

**Comment by arandr** [comment]
Manually merged this one, so it's closed. Don't worry about it, @darkracer.

**Pull request merged**